### PR TITLE
Adds test for MultipleEntryWithPredicateOperation&MultipleEntryWithPr…

### PR DESCRIPTION
**Pull request by ahmetmircik**
…edicateBackupOperation

### CODE REVIEW

File: hazelcast/src/test/java/com/hazelcast/map/EntryProcessorTest.java [hunk 1]
```diff
@@ -1192,6 +1193,72 @@ public Object process(Map.Entry entry) {
...
+        }
+
+        @Override
+        public Set<QueryableEntry> filter(QueryContext queryContext) {
+            return null;
+        }
+
+        @Override
+        public boolean isIndexed(QueryContext queryContext) {
+            return false;
```
Comment (eminn) [inline hazelcast/src/test/java/com/hazelcast/map/EntryProcessorTest.java]: shouldn't this method return `true` ?

Comment (ahmetmircik) [inline hazelcast/src/test/java/com/hazelcast/map/EntryProcessorTest.java]: fixed.

**Comment by eminn** [comment]
👍

**Comment by ahmetmircik** [comment]
Merging test fix.

**Pull request merged**